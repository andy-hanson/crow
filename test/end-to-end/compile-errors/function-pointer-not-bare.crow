main void()
	f void function() = &foo
	f[]

in-js void() js extern
	f void function() = &foo
	f[]

foo void()
	info log "in foo"
