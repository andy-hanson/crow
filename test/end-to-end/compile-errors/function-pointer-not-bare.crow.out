exit code: 0
stdout:
info: in foo
stderr:
test/end-to-end/compile-errors/function-pointer-not-bare.crow 2:25-2:29 The target of a function pointer must be a 'bare' function.
